docs/eventing/broker/alternate/README.md

```diff
@@ -6,3 +6,7 @@ implementations provided by Knative Eventing.
 
 ## GCP Broker
 The GCP Broker is optimized for running in GCP. For more details, refer to the [doc](https://github.com/google/knative-gcp/blob/master/docs/install/install-gcp-broker.md).
+
+## Apache Kafka Broker
+
+The Kafka Broker is an alternative Kafka Broker. For more details, refer to the [doc](../kafka-broker.md).
```

Comment:
```suggestion
The Apache Kafka Broker is an alternative Kafka Broker. For more details, refer to the [doc](../kafka-broker.md).
```
Either that or The Apache Kafka Broker is an alternative Broker. It's under the alternate page so it could also just be a description like the GCP one above it.

Reply:
I chose to keep it generic following the suggestion here https://github.com/knative/docs/pull/2740#discussion_r469375093 and I'll add more information to the linked page. WDYT?